### ISSUE TITLE
Can you reproduce the psnr?

### ISSUE THREAD

**Issue by jybbjybb** [Lornatang/ESRGAN-PyTorch#24]
I follow the instructions and try hyper-parameter tuning. 100, 200, 300 epochs, learning rate 2e-4, image_size 128 or 192, bs=16,32, 64. I can only get 31.7 dB PSNR on SET5 testset, still 1dB from reported in this repo. Can someone reproduce this 32.7dB on SET5?

**Comment by jybbjybb** [comment]
By the way, I use DIV2K (800 image) + Flickr (~2600 images).

**Comment by Lornatang** [comment]
It is ok to add OST data

**Comment by Lornatang** [comment]
The reproduction environment is：
CUDA: 11.5
CUDNN: 8.3.3
PyTorch: 1.11.0+cu115
GPU: RTX 3090
seed: 0
